BarcodeScanner.xcodeproj/project.pbxproj

```diff
@@ -228,7 +234,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
```

Comment:
Could it be `9.0` or there are some specific APIs you want to use? @chadcummings

Reply:
After reviewing which devices can run iOS 9.0, I discovered that all of the target devices can also run iOS 9.3. This led me to the conclusion that is was pointless to support iOS 9.0 when all devices that run iOS 9.0 will run iOS 9.3. Supporting iOS 9.0 has no benefit, as no additional devices are gained. That's my only reasoning, really.

Ultimately, this code is your's. I am happy to use whichever version you would like, but I would prefer trimming extra baggage as much as possible.

Reply:
I totally agree with you, but I think we still have projects with minimum target set to 9.0, which is probably something we might want to change. What do you think @onmyway133 @zenangst?

Reply:
I don't have strong opinion about this. But as there's no reason to raise to 9.3, then 9.0 is fine I think

Reply:
Sounds good. I'll move it to 9.0.

Reply:
@vadymmarkov yeah we should totally change that. But if we don't need `9.3` changes in this PR then it should still use `9.0` :)